### PR TITLE
Added troubleshooting topics for Orphaned cVMS

### PR DESCRIPTION
**Pull request by vidya-v**
VIC Appliance Checklist:
- [ ] Up to date with `master` branch
- [ ] Added tests
- [ ] Considered impact to upgrade
- [ ] Tests passing
- [ ] Updated documentation
- [ ] Impact assessment checklist

If this is a feature or change to existing functionality, consider areas of impact with the [Impact
Assessment Checklist](https://github.com/vmware/vic-product/blob/master/installer/docs/CHANGE.md)

Fixes #

### CODE REVIEW

**Comment by vidya-v** [comment]
Hi @stuclem, I have added he troubleshooting information per https://github.com/vmware/vic/issues/8015#issuecomment-395522974 
Request your review. Thanks!

File: docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Container VMs Orphaned After Upgrade
+
+After you upgrade vSphere Integrated Containers from version  1.2.x or 1.3.x to 1.4.0, associations between container VMs and virtual container hosts (VCH) are broken and the container VMs are orphaned. When you try to manage the conainer VMs through the Docker client, the container VMS are not listed.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md]: 1.2.x or 1.3.x to 1.4.0 ==> 1.2.x or 1.3.x to 1.4.0 or later
container VMs are orphaned ==> appear as orphaned in the vSphere Client
typo: conainer

File: docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+## Problem
+
+After an upgrade, when you run `docker container ls` on the Docker client, the container VMs might not be listed. 
+
+## Cause
+
+This error can occur in one the following situations:
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md]: one of

File: docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+* The container VMS that are not listed were missing before the upgrade. 
+* The container VMS are not in the VM folder that is created on the VCH during the upgrade procedure.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md]: VM folder that is created on the VCH ==> VM folder that is created in the image store

File: docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+* The container VMs and the VCH Endpoint VM are already moved to the container VM before the upgrade.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md]: moved to the container VM ==>  moved to the container VM folder

File: docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## Solution
+
+If the container VMs that are not listed on the Docker client are present in vSphere, verify if they are in the VM folder on VCH after the upgrade. If they are not present, then manually move the container VMs into that folder.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md]: are present in vSphere ==> are present in the vSphere Client

in the VM folder on VCH ==> in the VM folder in the image store

Comment (vidya-v) [reply docs/user_doc/vic_vsphere_admin/ts_orphaned_cvms.md]: Thanks, @stuclem . I will make the changes.

**Comment by vidya-v** [comment]
Hi @stuclem ,  I have made the changes.

**Comment by vidya-v** [comment]
Thanks, @stuclem !
@hickeng , I have added troubleshooting information per [vmware/vic#8015 (comment)](https://github.com/vmware/vic/issues/8015#issuecomment-395522974)
Request your review. 
Thanks!

**Pull request merged**